clients/java/src/main/java/io/camunda/zeebe/client/api/command/DeployProcessCommandStep1.java

```diff
@@ -15,87 +15,89 @@
  */
 package io.camunda.zeebe.client.api.command;
 
+import io.camunda.zeebe.client.api.command.DeployCommandStep1.DeployCommandStep2;
 import io.camunda.zeebe.client.api.response.DeploymentEvent;
 import io.camunda.zeebe.model.bpmn.BpmnModelInstance;
 import java.io.InputStream;
 import java.nio.charset.Charset;
 
+/** @deprecated for removal since 8, replaced by {@link DeployCommandStep1} */
```

Comment:
:wrench: Should we add when it will be removed, i.e. 8.1?

Reply:
Is that common? AFAIK, deprecation messages generally only state: since when, whether it's expected to be removed, and what to do about it. If you want I can add it. Do you think we should?

Reply:
As a user, I would like to know when is something going to be removed. Of course, we could have it project wide in the docs - it's possible we already have it and I'm unaware of it? But I don't think we have this specified anywhere, so how would a user know how long they have before it's removed? As a user, what is a better UX for you?